sys/fuchsia/fidl_process.txt

```diff
@@ -1,13 +1,90 @@
-# Copyright 2018 syzkaller project authors. All rights reserved.
-# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+# Copyright 2018 The Fuchsia Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+#
+# WARNING: This file is machine generated by fidlgen.
 
-include <zircon/syscalls.h>
+fuchsia_process_LauncherLaunchRequest {
+	txid		int32[0]
```

Comment:
I think we need a separate template type for the message header (4 first fields) in fidl.txt, it should be parametrized by inline/outofline data types and ordinal value.

Reply:
I separated out message header as a separate struct. However, I did not parameterize inline/outofline data, because it seemed more readable to me if we have separate request struct for each method.